.github/workflows/test.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
     - uses: twisted/python-info-action@v1
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip tox coverage coveralls
+        python -m pip install --upgrade pip tox coverage
```

Comment:
Make this change in the release branch itself is confusing.  Can you point to the discussion of the motivation for this change?

Reply:
I will revert. there was no discussion.

Just coverage reporting failing... so I took a look at the code and saw that codecov v1 was deprecated and no reports were generated for coveralls